tasks/collect-data/tests/test-collect-data-with-extra-data.yaml

```diff
@@ -96,8 +91,6 @@ spec:
           value: default/releaseplan-sample
         - name: releaseplanadmission
           value: default/releaseplanadmission-sample
-        - name: releasestrategy
-          value: default/releasestrategy-sample
```

Comment:
It won't let me comment on it, but new lines 105-106 and the check-result task aren't right. They still refer to extraData instead of data